src/renderer.js

```diff
@@ -22,6 +22,26 @@ function stripPage() {
   elements.forEach((e) => e.remove());
 }
 
+/**
+ * Injects a <base> tag which allows other resources to load. This
+ * has no effect on serialised output, but allows it to verify render quality.
+ * @param {string} url - Requested URL to set as the base.
+ */
+function injectBaseHref(url) {
+  const base = document.createElement('base');
+  base.setAttribute('href', url);
+  document.head.appendChild(base);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base Only the first href and first target are used. As we're using appendChild, we'll always preference any existing <base href> tags.